src/main/java/ca/corefacility/bioinformatics/irida/processing/impl/FastqcFileProcessor.java

```diff
@@ -122,7 +127,7 @@ private void processSingleFile(SequenceFile sequenceFile) throws FileProcessorEx
 
 			AnalysisFastQC analysisFastQC = analysis.build();
 
-			sequenceFile.setFastQCAnalysis(analysis.build());
+			sequenceFile.setFastQCAnalysis(analysisFastQC);
```

Comment:
Analysis was being built twice so I just used the result of the first build. The rest of the changes in the file are due to the formatter.